docs/running-tests-linux-locally.md

```diff
@@ -0,0 +1,39 @@
+# Running Tests on Linux Locally
+
+There are currently two ways to run tests on Linux locally:
+
+- Using WSL (Windows Subsystem for Linux)
+- Using Docker
```

Comment:
we need some extra knowledge on how to install docker - Actually copy pasting what you already wrote for autoscan I suppose?